planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
@kiuby88, what do I think? :)
I think you guys should close #252 #257 and #259 and open a new PR that changes consistently the DAMGenerator to do whatever you want to achieve. My two cents